### PR TITLE
Pydocstyle Errors

### LINKED ISSUE #199: pydocstyle errors
Errors raised in jobs run in Github actions. `poetry run pydocstyle tests/ toqito/ || true`

```python
toqito/nonlocal_games/xor_game.py:177 in public method `quantum_value`:
        D206: Docstring should be indented with spaces, not tabs
toqito/nonlocal_games/xor_game.py:177 in public method `quantum_value`:
        D207: Docstring is under-indented
toqito/nonlocal_games/xor_game.py:245 in public method `classical_value`:
        D202: No blank lines allowed after function docstring (found 1)
toqito/channel_metrics/completely_bounded_trace_norm.py:11 in public function `completely_bounded_trace_norm`:
        D205: 1 blank line required between summary line and description (found 0)
toqito/matrix_ops/inner_product.py:1 at module level:
        D400: First line should end with a period (not 'n')
toqito/matrix_ops/inner_product.py:6 in public function `inner_product`:
        D[20](https://github.com/purva-thakre/toqito/actions/runs/6527284296/job/17721926644#step:7:21)2: No blank lines allowed after function docstring (found 1)
toqito/matrix_ops/outer_product.py:1 at module level:
        D400: First line should end with a period (not 'n')
toqito/matrix_ops/outer_product.py:6 in public function `outer_product`:
        D209: Multi-line docstring closing quotes should be on a separate line
toqito/state_opt/state_exclusion.py:102 in private function `_min_error_primal`:
        D401: First line should be in imperative mood; try rephrasing (found 'The')
toqito/state_opt/state_exclusion.py:1[29](https://github.com/purva-thakre/toqito/actions/runs/6527284296/job/17721926644#step:7:30) in private function `_min_error_dual`:
        D401: First line should be in imperative mood; try rephrasing (found 'The')
toqito/matrix_props/trace_norm.py:6 in public function `trace_norm`:
        D205: 1 blank line required between summary line and description (found 0)
toqito/matrix_props/trace_norm.py:6 in public function `trace_norm`:
        D400: First line should end with a period (not '_')
toqito/matrix_props/is_diagonally_dominant.py:8 in public function `is_diagonally_dominant`:
        D208: Docstring is over-indented
```

### PR DESCRIPTION
**Pull request by purva-thakre**
Fixes #199

### CODE REVIEW

**Comment by purva-thakre** [comment]
@vprusso This one is ready for a review. 

BTW you might want to change the merge conditions (in project settings) to at least one approval before merging a PR.

File: toqito/channel_metrics/completely_bounded_trace_norm.py
Comment (purva-thakre) [inline toqito/channel_metrics/completely_bounded_trace_norm.py:15]:
```suggestion
    channel [WatCNorm18]_. The algorithm in p.11 of [WatSDP09] with
```
might want to add this space here.

**Review by vprusso — APPROVED** [review]
Nice one! LGTM! Merge when ready!

**Pull request merged**
